src/main/java/eu/execom/pomodoroTeam/entities/UserEntity.java

```diff
@@ -32,13 +27,11 @@
     @Column(nullable = false)
     private String email;
 
-    @OneToMany(mappedBy = "user", fetch = FetchType.LAZY, cascade = {CascadeType.REFRESH})
+    @OneToMany(mappedBy = "user")
     private List<PomodoroEntity> pomodoros = new ArrayList<>();
 
-    @ManyToMany(cascade = CascadeType.REFRESH)
-    @JoinTable(name = "User_Team", joinColumns = {
-            @JoinColumn(name = "User_id", nullable = false, updatable = false)}, inverseJoinColumns = {
-                    @JoinColumn(name = "Team_id", nullable = false, updatable = false)})
-    private List<TeamEntity> teams = new ArrayList<>();
+    @ManyToMany(mappedBy = "user")
+    @JsonBackReference
+    private List<TeamEntity> team = new ArrayList<>();

```

Comment:
There are multiple teams in a list, therefore the field should be named `teams`.